RepresentsMe/Sources/Utilities/UIViewControllerExt.swift

```diff
@@ -44,4 +44,23 @@ extension UIViewController {
         // Present the alert
         self.present(alert, animated: true, completion: completion)
     }
+    
+    /// Presents an alert specific to when an event is exported. Takes user to the
+    /// calendar app if they so desire
+    ///
+    /// - Parameter date:      the date at which the exported event is taking place.
+    func exportEventAlert(date: Date) {
```

Comment:
Since this is specific to exporting events can we move this to the event details view controller?